RFC.md

```diff
@@ -1,19 +1,171 @@
 # RFC: Amplify JS TypeScript Improvements
 
 # General TypeScript Improvements
+
 - Version upgrade
 - Strict mode
 
 # Utility Changes
-- Hub typing
-- Configuration
+
+Amplify is proposing the following changes for our core utilities.
+
+## TypeScript support for Amplify Hub channels
+
+We are improving **_DX_** by adding strict type support to Hub channels, events, and payloads. An example of the developer experience when listening for `auth` events is highlighted below.
+
+**Amplify v5 & v6 (`aws-amplify@^5`)**
```

Comment:
Why `& v6`?

Reply:
because API in v5 and v6 are the same, what is changing is DX

Reply:
This was feedback I provided in my review, in this case the example code is identical for both v5 & v6, the only difference is type support. I figured we could have sample code + screen shots to show the improved intellisense.